packages/cli/src/databases/migrations/mysqldb/1690000000001-MigrateIntegerKeysToString.ts

```diff
@@ -0,0 +1,258 @@
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}shared_workflow DROP COLUMN \`tmp_workflowId\`;`,
+		);
+
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_statistics RENAME COLUMN \`workflowId\` to \`tmp_workflowId\`;`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_statistics ADD COLUMN \`workflowId\` varchar(36) NOT NULL;`,
+		);
+		await queryRunner.query(
+			`UPDATE ${tablePrefix}workflow_statistics SET \`workflowId\` = CONVERT(\`tmp_workflowId\`, CHAR);`,
+		);
+		await queryRunner.query(
+			`CREATE INDEX \`idx_workflow_statistics_workflow_id\` ON ${tablePrefix}workflow_statistics (\`workflowId\`);`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_statistics DROP FOREIGN KEY \`workflow_statistics_ibfk_1\`;`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_statistics ADD CONSTRAINT \`fk_workflow_statistics_workflow_id\` FOREIGN KEY (\`workflowId\`) REFERENCES workflow_entity(id) ON DELETE CASCADE ON UPDATE NO ACTION;`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_statistics DROP CONSTRAINT \`PRIMARY\`, ADD PRIMARY KEY (\`workflowId\`, \`name\`);`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}workflow_statistics DROP COLUMN \`tmp_workflowId\`;`,
+		);
+
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}webhook_entity RENAME COLUMN \`workflowId\` to \`tmp_workflowId\`;`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}webhook_entity ADD COLUMN \`workflowId\` varchar(36) NOT NULL;`,
+		);
+		await queryRunner.query(
+			`UPDATE ${tablePrefix}webhook_entity SET \`workflowId\` = CONVERT(\`tmp_workflowId\`, CHAR);`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}webhook_entity DROP COLUMN \`tmp_workflowId\`;`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}webhook_entity ADD CONSTRAINT \`fk_webhook_entity_workflow_id\` FOREIGN KEY (\`workflowId\`) REFERENCES workflow_entity(id) ON DELETE CASCADE ON UPDATE NO ACTION;`,
+		);
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}execution_entity RENAME COLUMN \`workflowId\` to \`tmp_workflowId\`;`,
+		);
+
+		await queryRunner.query(
+			`ALTER TABLE ${tablePrefix}execution_entity ADD COLUMN \`workflowId\` varchar(36);`,
+		);
+		await queryRunner.query(
+			`UPDATE ${tablePrefix}execution_entity SET \`workflowId\` = CONVERT(\`tmp_workflowId\`, CHAR);`,
+		);
+		await queryRunner.query(
+			`CREATE INDEX \`idx_execution_entity_workflow_id_id\` ON ${tablePrefix}execution_entity (\`workflowId\`,\`id\`);`,
+		);
+		await queryRunner.query(
+			`CREATE INDEX \`idx_execution_entity_workflow_id_finished_id\` ON ${tablePrefix}execution_entity (\`workflowId\`,\`finished\`,\`id\`);`,
+		);
+		await queryRunner.query(
+			`CREATE INDEX \`idx_execution_entity_workflow_id_wait_till_id\` ON ${tablePrefix}execution_entity (\`workflowId\`,\`waitTill\`,\`id\`);`,
+		);
```

Comment:
Those indices should no longer be necessary as we now query the `status` column, so they can potentially be removed.

Reply:
ok, removing them